.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,18 @@
+### Links to any relevant github issues:
+
+- [Issue description](http://example.com/org/repo/issues/411). Any other useful information.
+
+### What's in the the PR
+
+- [ ] A description of the changes proposed in the pull request.
+  - [ ] Any areas or issues reviewer should pay attention to? (comments in diff okay).
+- [ ] Update `/docs` to include, whenever possible, a new, suitable recommendation about how to use
+    the feature.
+- [ ] Extra Credit: [Confirm it runs and tests pass on the Rubies specified in the Travis
```

Comment:
What's the real meaning of this sentence? Should contributors make sure "locally" that it runs on all rubies? Should they simply check that Travis CI tests pass? It really isn't clear to me what any contributor is supposed to do for this step.

And the `.travis.yml` link is not working. :wink: